doc/man7/EVP_KDF-X942-CONCAT.pod

```diff
@@ -0,0 +1,35 @@
+=pod
+
+=head1 NAME
+
+EVP_KDF-X942-CONCAT - The X942 Concat EVP_KDF implementation
+
+=head1 DESCRIPTION
+
+The EVP_KDF-X942-CONCAT algorithm is identical to EVP_KDF-X963. It is
+used for Keyagreement to derive a key using input such as a shared secret key
```

Comment:
Keyagreement should be uncapitalised and two words.

Reply:
Amusingly enough, the bit in the KeyUsage extension is named "keyAgreement"